include/swift/AST/AutoDiff.h

```diff
@@ -354,6 +354,13 @@ class AutoDiffIndexSubset : public llvm::FoldingSetNode {
   unsigned getNumIndices() const {
     return (unsigned)std::distance(begin(), end());
   }
+  
+  SmallBitVector getBitVector() const {
```

Comment:
cc @rxwei

This is the only location, but when I asked Richard he recommended to add that into the API of IndexSubset. I'm guessing this may be used in the future in the other PR for the autodiff index subset refactoring?

Reply:
Yep. +1 to adding this method.